services/web-development.html

```diff
@@ -0,0 +1,191 @@
+---
+permalink: /services/web-development
+layout: default
+title: From MVP to Сomplete Product. Web Development Services | Active Bridge
+description: Enhance your business processes leveraging our web application
+             development services. We excel at crafting digital experiences that
+             both look and work impressive.
+---
+
+<div class="container web_dev_services development">
+  <div class="web_dev_services__main wrap">
+    <div class="row">
+      <div class="header-2">{{ site.data.web_dev_services.title }}</div>
+      <div class="description">{{ site.data.web_dev_services.description }}</div>
+      <div class="web_dev_services__list">
+        <div class="header-3">{{ site.data.web_dev_services.expertise.title }}</div>
+        <div class="web_dev_services__list-wrap">
+          <img src="/assets/images/web_development_services/star.webp" alt="Star">
+          <ul class="web_dev_services__list-text">
+            {% for item in site.data.web_dev_services.expertise.all %}
+              <li class="web_dev_services__list-item">{{ item.item }}</li>
+            {% endfor %}
+          </ul>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <div class="web_dev_services__web_solutions wrap">
+    <div class="row">
+      <div class="header-2">{{ site.data.web_dev_services.web_solutions.title }}</div>
+      <div class="web_dev_services__list">
+        <div class="header-3">
+          {{ site.data.web_dev_services.web_solutions.subtitle_1 }}
+        </div>
+        <div class="web_dev_services__list-wrap">
+          <img src="/assets/images/web_development_services/branded_cloud.webp" alt="Cloud">
+          <ul class="web_dev_services__list-text list_with_columns">
+            {% for item in site.data.web_dev_services.web_solutions.all %}
+              <li class="web_dev_services__list-item">{{ item.item }}</li>
+            {% endfor %}
+          </ul>
+        </div>
+      </div>
+      <div class="header-2">
+        {{ site.data.web_dev_services.web_solutions.subtitle_2 }}
+      </div>
+    </div>
+  </div>
+
+  <div class="web_dev_services__full_stack wrap">
```

Comment:
Please check mobile screen for this section
![image](https://github.com/katatsu12/live_blog/assets/22080708/74343c2a-efdf-4595-ab47-a8cb08466f39)